config/software/libsodium.rb

```diff
@@ -22,11 +22,18 @@
 license "ISC"
 license_file "LICENSE"
 
-dependency "autoconf"
-dependency "automake"
-dependency "libtool"
+# Depend on the msys2/mingw environment given to us and don't build our
+# own build tools on windows.
+unless windows?
+  dependency "autoconf"
+  dependency "automake"
+  dependency "libtool"
```

Comment:
@ksubrama does this assume that the toolchain version with these has been deployed? As it currently stands, it has not been deployed because adding autoconf breaks the chefdk build (libgecode depselector sadness). This is currently being worked on but it's not fixed yet...